Scripts/RemediationScripts/Remediate-RemoteDesktopAccess.ps1

```diff
@@ -0,0 +1,357 @@
+﻿<##########################################
+
+# Overview:
+    This script is used to disable remote Desktop (RDP) access on cloud service(s).
+
+ControlId: 
+    Azure_CloudService_SI_Disable_RemoteDesktop_Access
+
+DisplayName:
+    Remote Desktop (RDP) access must be disabled on cloud service roles.    
+
+# Pre-requisites:
+    You will need co-administrator role on subscription.
```

Comment:
Please confirm if just a `Contributor` role would suffice.

Reply:
We can execute remediation script using contributor permission, have added changes for this